node/risk-app/server/models/risk-view.js

```diff
@@ -95,9 +95,9 @@ function riskViewModel (risk, address, backLinkUri) {
     this.secondSource = 'surface-water.html'
   }
   if (config.riskPageFlag) {
-    this.firstSource = 'partials/flagged/' + this.firstSource
-    this.secondSource = 'partials/flagged/' + this.secondSource
-    this.additionalInformation = 'partials/flagged/groundwaterAndReservoirs.html'
+    this.firstSource = 'partials/' + this.firstSource
+    this.secondSource = 'partials/' + this.secondSource
+    this.additionalInformation = 'partials/groundwaterAndReservoirs.html'
   } else {
```

Comment:
This whole section needs to be removed and the `else` block be the only code that's needed. That config value no longer exists.